src/ssl.c

```diff
@@ -4992,6 +4992,9 @@ int AddCA(WOLFSSL_CERT_MANAGER* cm, DerBuffer** pDer, int type, int verify)
     #elif defined(SMALL_SESSION_CACHE)
         #define SESSIONS_PER_ROW 2
         #define SESSION_ROWS 3
+    #elif defined(VERY_SMALL_SESSION_CACHE)
+        #define SESSIONS_PER_ROW 2
+        #define SESSION_ROWS 1
```

Comment:
Non prime numbers do not work well with hashing.

Reply:
The idea behind this change is reduce memory due to the microcontroller only having 32KB of RAM.  Is this because of collisions in the hashing algorithm?  Would changing SESSIONS_PER_ROW to 1 help?

Reply:
Actually, just remembered i dont need this any more, so can remove it. Might just call the crypto functions directly for the server socket.

Reply:
Recommend removing this or adjusting SESSIONS_PER_ROW to prime like 3.